Library/Formula/owamp.rb

```diff
@@ -0,0 +1,49 @@
+class Owamp < Formula
+  desc "Implementation of the One-Way Active Measurement Protocol"
+  homepage "http://www.internet2.edu/performance/owamp/"
```

Comment:
It looks like this more or less redirects to `http://software.internet2.edu/owamp/` now, so perhaps worth using that?